pandas/core/arrays/sparse/array.py

```diff
@@ -1392,6 +1392,30 @@ def mean(self, axis=0, *args, **kwargs):
             nsparse = self.sp_index.ngaps
             return (sp_sum + self.fill_value * nsparse) / (ct + nsparse)
 
+    def max(self, axis=0, *args, **kwargs):
+        nv.validate_max(args, kwargs)
+
+        if self.sp_index.ngaps > 0 and np.all(self._valid_sp_values < 0):
+
+            if self.size > 0 and self._valid_sp_values.size == 0:
+                return np.nan
+
+            return 0
+        else:
+            return np.amax(self._valid_sp_values, axis)
```

Comment:
any particular reason for np.amax instead of np.max?

Reply:
The explanation documentation for np.amax is much more comprehensive than np.max. This is so that where other people read the code, they can quickly refer to the documentation rather than the reader requiring to go to the np.max page than the np.amax page.